packages/shared/src/components/modals/common/ModalFooter.tsx

```diff
@@ -5,5 +5,9 @@ export type ModalFooterProps = {
 };
 
 export function ModalFooter({ children }: ModalFooterProps): ReactElement {
-  return <footer>{children}</footer>;
+  return (
+    <footer className="flex gap-3 justify-end items-center py-4 px-2 w-full h-14 border-t border-theme-divider-tertiary">
```

Comment:
I see the justify-end here, but not sure that's the case with multiple ones right?
Also pretty sure I know the answer, does every footer have a border?

Reply:
Footer seems to be right-aligned:
https://www.sketch.com/s/6d2c8a99-2779-4674-87f4-43717dc23eec/a/EL3nQdO

Reply:
Yes I see, just the one that has it centered (but see you raised this already)
And the markdown footer that's different right?

Reply:
Yeap, I am workin on it as part of the header tabs